source/_integrations/lg_thinq.markdown

```diff
@@ -0,0 +1,126 @@
+- [Dryer](https://thinq.developer.lge.com/en/cloud/docs/thinq-connect/device-profile/Dryer/)
+- [Home Brew](https://thinq.developer.lge.com/en/cloud/docs/thinq-connect/device-profile/Home-Brew/)
+- [Hood](https://thinq.developer.lge.com/en/cloud/docs/thinq-connect/device-profile/Hood/)
+- Humidifier
+- [Kimchi Refrigerator](https://thinq.developer.lge.com/en/cloud/docs/thinq-connect/device-profile/Kimchi-Refrigerator/)
+- [Microwave Oven](https://thinq.developer.lge.com/en/cloud/docs/thinq-connect/device-profile/Microwave-Oven/)
+- [Oven](https://thinq.developer.lge.com/en/cloud/docs/thinq-connect/device-profile/Oven/)
+- [Plant Cultivator](https://thinq.developer.lge.com/en/cloud/docs/thinq-connect/device-profile/Plant-Cultivator/)
+- [Refrigerator](https://thinq.developer.lge.com/en/cloud/docs/thinq-connect/device-profile/refrigerator/)
+- [Robot Cleaner](https://thinq.developer.lge.com/en/cloud/docs/thinq-connect/device-profile/robot-cleaner/)
+- [Stick Cleaner](https://thinq.developer.lge.com/en/cloud/docs/thinq-connect/device-profile/Stick-Cleaner/)
+- [Styler](https://thinq.developer.lge.com/en/cloud/docs/thinq-connect/device-profile/styler/)
+- [System Boiler](https://thinq.developer.lge.com/en/cloud/docs/thinq-connect/device-profile/System-Boiler/)
+- Washcombo Main
+- Washcombo Mini
+- [Washer](https://thinq.developer.lge.com/en/cloud/docs/thinq-connect/device-profile/washer/)
+- [Washtower](https://thinq.developer.lge.com/en/cloud/docs/thinq-connect/device-profile/WashTower-Single-Unit/)
+- [Washtower Dryer](https://thinq.developer.lge.com/en/cloud/docs/thinq-connect/device-profile/WashTower-Dryer/)
+- [Washtower Washer](https://thinq.developer.lge.com/en/cloud/docs/thinq-connect/device-profile/WashTower-Washer/)
+- [Water Heater](https://thinq.developer.lge.com/en/cloud/docs/thinq-connect/device-profile/Water-Heater/)
+- [Water Purifier](https://thinq.developer.lge.com/en/cloud/docs/thinq-connect/device-profile/water-purifier/)
+- [Wine Cellar](https://thinq.developer.lge.com/en/cloud/docs/thinq-connect/device-profile/Wine-Cellar/)
+
+
+## Troubleshooting
+
+### Setup
+
+#### Aborted: The token is not valid
+
+This error occurs when the Personal Access Token (PAT) is invalid or entered incorrectly. Please visit the [Personal Access Token Page](https://d1jykc6oogauei.cloudfront.net/) page to check if your token is valid.
+
+#### Aborted: The country is not supported
+
+Check your PAT's valid country in the **Choose the region / Country** section.
+
+#### Error: The number of API calls has been exceeded
+
+This error occurs when there's an abnormal number of API calls made using the PAT.
+
+
+## Debugging
+
+The LG ThinQ integration will show additional information by enabling log configuration. Add the line to your {% term "`configuration.yaml`" %}:
+
+```yaml
+logger:
+  default: info
+  logs:
+    homeassistant.components.lg_thinq: debug
+```
```

Comment:
maybe we should add a step to remove the debug level once they are done debugging?

Reply:
Do you mean we need to mention how to remove the debug level?
It is just removing the relevant line from the configuration.yaml.

Reply:
@c0ffeeca7 
How can I resolve this one?
I can't find any other integrations to refer.
We just referred other integrations like smartthings.